U/Unidecode/Compat.toml

```diff
@@ -1,2 +1,5 @@
-["1.0"]
+["1.0.0-1.0.1"]
 julia = "1.0-1.3"
+
+["1.0.2"]
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.